system-addon/lib/ActivityStream.jsm

```diff
@@ -81,6 +82,21 @@ const PREFS_CONFIG = new Map([
       "provider_name": "Pocket",
       "provider_icon": "pocket"
     }`
+  }],
+  ["migrationExpired", {
+    title: "Boolean flag that will always be true once the migration period has passed.",
+    value: false,
+    value_local_dev: false
+  }],
+  ["migrationRemainingDays", {
+    title: "Number of days to show the manual migration message",
+    value: 4,
+    value_local_dev: 2
+  }],
+  ["migrationLastShownDate", {
+    title: "Timestamp when migration message was last shown",
+    value: 0,
+    value_local_dev: 0
```

Comment:
Any particular reason you have `value_local_dev` for each of these? If it's the same as `value`, then it's not necessary. And I don't think you need a custom value for developers here.